### PR TITLE
[ISSUE 754] Define listing of Acls API

### PR DESCRIPTION
**Pull request by jonasgeiregat**
- [x] **Breaking change?** (if so, please describe the impact and migration path for existing applications:)

**What changes did you make?** (Give an overview)

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "X" next to an item)

- [x] No need to
- [x] Manually(please, describe, when necessary)
- [x] Unit checks
- [x] Integration checks
- [x] Covered by existing automation

**Checklist** (put an "X" next to an item, otherwise PR will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings(e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

### CODE REVIEW

**Review by Haarolean — CHANGES_REQUESTED** [review]
@jonasgeiregat can we return whole `AclBinding` instead of `AccessControlEntry`?

**Comment by jonasgeiregat** [comment]
> @jonasgeiregat can we return whole `AclBinding` instead of `AccessControlEntry`?

You mean instead of `/api/clusters/{clusterName}/acls/{resourceType}` just the list of all Acls per cluster like `/api/clusters/{clusterName}/acls` ?

**Comment by Haarolean** [comment]
> > @jonasgeiregat can we return whole `AclBinding` instead of `AccessControlEntry`?
> 
> You mean instead of `/api/clusters/{clusterName}/acls/{resourceType}` just the list of all Acls per cluster like `/api/clusters/{clusterName}/acls` ?

You're returning a list of `AccessControlEntry`, but I'd like to see `AclBinding`, which is a parent entity.

File: kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml [hunk 1]
```diff
@@ -2582,6 +2620,23 @@ components:
...
+        host:
+          type: string
+        operation:
+          type: string
+          enum: [UNKNOWN, ANY, ALL, READ, WRITE, CREATE, DELETE, ALTER, DESCRIBE, CLUSTER_ACTION, DESCRIBE_CONFIGS, ALTER_CONFIGS, IDEMPOTENT_WRITE]
+        permissionType:
+          type: string
+          enum: [UNKNOWN, ANY, DENY, ALLOW]
+      required:
+        - totalReplicationFactor
```
Comment (Haarolean) [inline kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml:2637]: What's that?

**Review by Haarolean — CHANGES_REQUESTED** [review]

**Pull request closed without merging**